lib/serverengine/server.rb

```diff
@@ -66,12 +66,14 @@ def install_signal_handlers
       s = self
       SignalThread.new do |st|
         st.trap(Daemon::Signals::GRACEFUL_STOP) { s.stop(true) }
-        st.trap(Daemon::Signals::IMMEDIATE_STOP) { s.stop(false) }
-        st.trap(Daemon::Signals::GRACEFUL_RESTART) { s.restart(true) }
-        st.trap(Daemon::Signals::IMMEDIATE_RESTART) { s.restart(false) }
-        st.trap(Daemon::Signals::RELOAD) { s.reload }
         st.trap(Daemon::Signals::DETACH) { s.stop(true) }
-        st.trap(Daemon::Signals::DUMP) { Sigdump.dump }
+        unless $platformwin
```

Comment:
There is no command like kill in unix. 
Then, to stop ServerEngine, we should use Ctrl+C in command prompt to send SIGTERM.
(It is impossible to install a command called kill, but it is bored and there is only SIGTERM, so it doesn't make any sense.)